test/BasketHandlerEdgeCases.test.ts

```
@@ -0,0 +1,43 @@
+    });
+
+    basketHandler = await BasketHandler.deploy();
+    await basketHandler.deployed();
+  });
+
+  describe("Critical Edge Case Validations", function () {
+    it("1. Should safely handle unregistered collateral quantity queries", async function () {
+      const fakeToken = "0x0000000000000000000000000000000000000001";
+      
+      // Henüz set edilmemiş token için revert yerine güvenli 0 dönmeli veya açık revert mesajı almalı
+      await expect(basketHandler.quantity(fakeToken)).to.be.reverted;
```

ESLint

[error] Delete `;`

(prettier/prettier)